src/main/scala/uk/gov/hmrc/initrepository/Main.scala

```diff
@@ -95,18 +96,21 @@ object Main {
         } else {
           root.setLevel(Level.INFO)
         }
-        start(config.repoName, config.teamName, config.repoType)
+
+        val webhookUrl = Properties.envOrNone("webhookUrl")
```

Comment:
@agilestevesmith @charleskubicek I thought by having it as an env var it can be optional. Should I think of a way to make the command line parameter optional?